### ISSUE TITLE
[servicebus] why messages received from service bus sdk with unexpected prefix before json data 

### ISSUE THREAD

**Issue by montaler** [Azure/azure-sdk-for-go#18547]
### Bug Report
- package: https://github.com/Azure/azure-sdk-for-go/tree/main/sdk/messaging/azservicebus
- SDK version : v1.0.0
- go version: 1.18.3

- What happened?
   I am trying use sdk to receive service bus message and json.Unmarshal it to struct, but I found that the return message with unexpected prefix `@strin2http://schemas.microsoft.com/2003/10/Serialization/` before real json data.
  - For example unexpected message:
     @strin2http://schemas.microsoft.com/2003/10/Serialization/   {"id":"xxxx-xxxx"}
     the message will result in json.Unmarshal error. error message: invalid character '@' looking for beginning of value.
- What did you expect or want to happen?
   the message without unexpected prefix string
- How can we reproduce it?
   leverage sdk to receive service bus
- My code is the same with: 
   https://github.com/Azure/azure-sdk-for-go/blob/ad02514fbe2cd222430fa12a18c28b9c91b2dc08/sdk/messaging/azservicebus/example_receiver_test.go#L94

**Comment by montaler** [comment]
Hi @richardpark-msft, 
the sender is azure devops service hooks https://docs.microsoft.com/en-us/azure/devops/service-hooks/overview?view=azure-devops, could you help to contact azure devops team to upgrade the **older .NET Service Bus packages** ?

**Comment by richardpark-msft** [comment]
That is interesting, I'm guessing there's a backwards compatibility issue with changing it by default.

I did find this page, which I believe talks about that task (apologies, as Azure DevOps is far outside of my wheelhouse):
https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/utility/publish-to-azure-service-bus?view=azure-devops#open-source

That page does link to their repo, which should notify the team you're interested in (or, at the very least, provide you a better communication point than I can):
https://github.com/Microsoft/azure-pipelines-tasks/tree/master/Tasks/PublishToAzureServiceBusV1

It might be worthwhile to post a question there to see if they have a way to avoid doing the binary encoding that I mentioned above. 

Closing as this is not an Azure SDK issue, specifically.
